src/xrpld/app/misc/NetworkOPs.cpp

```diff
@@ -1165,6 +1166,15 @@ NetworkOPsImp::submitTransaction(std::shared_ptr<STTx const> const& iTrans)
         return;
     }
 
+    // Enforce Network bar for batch txn
+    if (iTrans->isFlag(tfInnerBatchTxn) &&
+        m_ledgerMaster.getValidatedRules().enabled(featureBatch))
```

Comment:
Seeing a codecov issue here - is this possible to hit?

Reply:
added: https://github.com/XRPLF/rippled/pull/5060/commits/569ac2102e40da71d0e5216c50985f3c29cc6f67